Dockerfile

```diff
@@ -7,4 +7,6 @@ ADD . /go/src/github.com/redhatanalytics/oshinko-rest
 WORKDIR /go/src/github.com/redhatanalytics/oshinko-rest
 RUN make install
 
+RUN chmod a+rwX -R .
```

Comment:
'X' only sets execute on files that already have execute, which differs from 'x' that always sets execute